app/src/main/java/chat/rocket/android/activity/MainActivity.java

```diff
@@ -53,14 +53,14 @@ protected void onStart() {
 
   @Override
   protected void onResume() {
-    super.onResume();
     isForeground = true;
+    super.onResume();
   }
 
   @Override
   protected void onPause() {
```

Comment:
I think we may leave the `isForeground = false;` before the call to super here. Looks like a bit more safe.